setup.py

```diff
@@ -31,6 +31,7 @@
     maintainer="The RIPE Atlas team",
     maintainer_email="atlas@ripe.net",
     install_requires=[
+        "IPy",
         "python-dateutil",
```

Comment:
Right, I was wondering how we get away with this but then I realized it's a dependency on sagan so it will be installed. But you are right it should be there.